src/components/yusuf.js

```diff
@@ -0,0 +1,2 @@
+console.log("Hello, Yusuf P2!\n");
```

Comment:
I don't like the \n at the end of your log. Please remove it.

Reply:
My wife went out

Reply:
I mean wifi

Reply:
So I am using my phone

Reply:
Okay, sounds good.

Reply:
Looks good!!